server/test/services/export/ProgramJsonSamplerTest.java

```diff
@@ -80,79 +79,82 @@ public void setUp() {
 
   @Test
   public void samplesFullProgram() {
-    CfJsonDocumentContext json = programJsonSampler.getSampleJson(programDefinition);
+    String json = programJsonSampler.getSampleJson(programDefinition);
 
     String expectedJson =
-        "[ {\n"
-            + "  \"applicant_id\" : 123,\n"
-            + "  \"application\" : {\n"
-            + "    \"name\" : {\n"
-            + "      \"first_name\" : \"Homer\",\n"
-            + "      \"last_name\" : \"Simpson\",\n"
-            + "      \"middle_name\" : \"Jay\"\n"
+        "{\n"
+            + "  \"nextPageToken\" : null,\n"
+            + "  \"payload\" : [ {\n"
```

Comment:
`ProgramJsonSampler::getSampleJson()` now returns the json already wrapper, so we have to update the test.